### ISSUE TITLE
Hi there, how can we talk 1:1 about chromote?

### ISSUE THREAD

**Issue by nbolton** [blakjakau/chromote#2]
@blakjakau Hello, I'm from the Synergy team. I hope you don't mind me opening an issue for this (I couldn't find any contact details for you). Would you be interested in talking about chromote?

**Comment by blakjakau** [comment]
@nbolton yeah we can talk. Email me fakeemail@jakbox.net and I'll ping you back from a real address
